src/content_script/input_tools.css

```diff
@@ -0,0 +1,35 @@
+.sl-button-wrapper {
+  position: absolute;
+  z-index: 999999;
+}
+
+.sl-button {
+  cursor: pointer;
+  box-shadow: 0px 0px 10px 0px rgba(0, 0, 0, 0.3);
+  max-height: 30px;
+  max-width: 30px;
+  position: absolute;
+  top: 50%;

```

Comment:
the `top, left, transform` here is to center the logo right?

Reply:
yes, exactly